Makefile

```diff
@@ -17,4 +17,5 @@ update:
 	npm run fetch-data
 
 build-size:
-	npm run size
+	npm run build-size
```

Comment:
`build-size` here but `size-build` in package.json

Reply:
this is for `make s + <TAB> -> make start` as more used case :)